ui/src/features/Editor/components/Canvas/components/Nodes/BatchNode/index.tsx

```diff
@@ -1,9 +1,11 @@
 import { RectangleDashed } from "@phosphor-icons/react";
-import { NodeProps, NodeResizer } from "@xyflow/react";
-import { memo, useState } from "react";
+import { NodeProps, NodeResizer, useReactFlow } from "@xyflow/react";
+import { memo, useState, useCallback } from "react";
 
 import { Node } from "@flow/types";
 
+import useBatch from "../../../useBatch";
```

Comment:
In another PR after this pr is merged, can you move all the canvas's useXxxx, like useBatch, to the global hooks directory? They are starting to be needed in more far-away places than originally imagined, so to keep import statements clean and things consolidated